packages/flutter/lib/src/painting/inline_span.dart

```diff
@@ -175,6 +239,28 @@ abstract class InlineSpan extends DiagnosticableTree {
     return buffer.toString();
   }
 
+  /// Flattens the [InlineSpan] tree to a list of plain text strings and
```

Comment:
Did you mean to a `list of InlenSpanSemanticsInformation objects`?

Reply:
Yup, fixing that.